.github/CONTRIBUTING.md

```diff
@@ -5,4 +5,30 @@ feel it's ready to merge. Even if there's no real disagreement about a PR, at
 least one other person on the team needs to look over a PR before merging. The
 purpose of this review requirement is to ensure shared knowledge of the app and
 its changes and to take advantage of the benefits of working together changes
-without any single person being a bottleneck to making progress.
+without any single person being a bottleneck to making progress.  
+
+## Steps  
+
+1. Fork this repository.  
+2. Clone the repository by running this command:  
+    ```shell
+    $ git clone https://github.com/YOUR-GITHUB-USERNAME/playtime.git
+    ```  
+3. Add a remote upstream in order for git to know the original Playtime repository is:  
+    ```shell
+    $ git remote add upstream https://github.com/rubyforgood/playtime.git
+    ```  
+4. To make changes, create a new branch and try to name it with the change you want to make as follows:  
```

Comment:
I think this line is grammatically correct, but it's a little confusing. Maybe breaking it into two sentences would help? Something like this: "Create a new branch to make changes. The branch name should be relevant to your change (ex. `rubocop` or `setup-mailers`)."

Also, a style nitpick: could you keep your lines under 80 characters? It makes it easier to read/navigate on terminal-based editors.